webapp/publisher/views.py

```diff
@@ -28,12 +28,13 @@ def get_account_details():
     return render_template("publisher/account-details.html")
 
 
-@publisher.route("/charms")
+@publisher.route('/<regex("charms|bundles"):package_type>')
```

Comment:
```suggestion
@publisher.route('/charms')
@publisher.route('/bundles')
```

No need for regex for this.

Reply:
The filtering can be done directly in the template.